src/components/Table/Table.tsx

```diff
@@ -110,17 +117,28 @@ export const Table = chakra((props: React.PropsWithChildren<TableProps>) => {
       }
     }
 
+    const cellContent = (key: number, column: string | JSX.Element) => {
+      return windowDimensions.width <= breakpointMedium ? (
+        <>
+          <Box as="span">{columnHeaders[key]}</Box>
```

Comment:
If nothing else is added to these `Box`es, then I think having direct `<span>`s should be okay.